src/components/Chat/Chat.vue

```diff
@@ -209,6 +209,7 @@
           :message-text="$route.query.messageText"
           @message="onMessage"
           @esc="replyMessageId = -1"
+          :clearInputValueOnSend="this.$store.state.balance < 0.001 === false"
```

Comment:
compare other side with `>=` and use shorter variant

```suggestion
          :clearInputValueOnSend="this.$store.state.balance >= 0.001"
```

Reply:
rename the prop name by using snake-case, just to keep the same style in the codebase